Countries/ListViewController.swift

```diff
@@ -7,24 +7,16 @@
 
 import UIKit
 
-
 class ListViewController: UITableViewController {
     
-    enum TableConstants {
+    private enum TableConstants {
```

Comment:
note: I would suggest default name Constants, for such enums.

Reply:
ok